src/frontend/reducers/reducerFunctions.js

```diff
@@ -0,0 +1,59 @@
+import { calculateDiscount } from "../pages/discount";
+
+export function filteredPriceList(sortBy, productList) {
+    if (sortBy === "HIGH_TO_LOW") {
+        return [...productList].sort((a, b) => calculateDiscount(b.price, b.discount) - calculateDiscount(a.price, a.discount));
+    }
+    if (sortBy === "LOW_TO_HIGH") {
+        return [...productList].sort((a, b) => calculateDiscount(a.price, a.discount) - calculateDiscount(b.price, b.discount));
+    }
```

Comment:
if you are sorting by discount here, then name the case accordingly, like `"SORT_BY_DISC_HIGH_TO_LOW", since there can be another sorting case like sort by price or something else, then "HIGH_TO_LOW" won't be understandable

Reply:
actually, I'm sorting by discounted price, I'm calculating the final price of the product dynamically using actual price and discount %, and for that, I'm using calculateDiscount(), it'll return the final price of the product.